File: Avaliacao/q1_python/Notebook.py
from Produto import Produto

class Notebook(Produto):
    def __init__(self, nome, preco, processador, memoria):
        super().__init__(nome, preco)
        self.processador = processador
        self.memoria = memoria

    def mostra_info(self):
        super().mostra_info()
        print(f"Processador: {self.processador}")
        print(f"Memoria: {self.memoria}")

    def instalar_aplicativo(self):
        print("Instalando aplicativo...")

File: Avaliacao/q1_java/src/main/java/capacitacao/core/Notebook.java
package capacitacao.core;

public class Notebook extends Produto{
    private String processador;
    private String memoria;

    public Notebook(String nome, double preco, String processador, String memoria) {
        super(nome, preco);
        this.processador = processador;
        this.memoria = memoria;
    }
    
    @Override
    public void mostraInfo(){
        super.mostraInfo();
        System.out.println("Processador: " + this.processador);
        System.out.println("Memoria: " + this.memoria);
    }

    public void instalarAplicativo(){
        System.out.println("Instalando aplicativo...");
    }
}
